fs_attachment_s3_migration/models/ir_attachment.py

```
@@ -0,0 +1,305 @@
+                    expected_size,
+                )
+            except OSError as e:
+                _logger.debug(
+                    "Cannot verify existing file %s: %s, overwriting", path, e
+                )
+
+        try:
+            with fs.open(path, "wb") as f:
+                f.write(bin_data)
+        except OSError as e:
+            _logger.error("Failed to write file %s: %s", path, e)
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)